### ISSUE TITLE
events (as function) behaving not correct on the depending view

### ISSUE THREAD

**Issue by Schwitzhoden** [fullcalendar/fullcalendar-scheduler#440]
See jsbin. Open the output window to see the description

https://jsbin.com/jipozedohu/1/edit?html,output

Problem is: If you load the year view, then switch back to the month view and manually reload the events for the monthview, the parameters 'start' and 'end'-momentObjects are still in the 'year'-mode and starts from YYYY-1-1 til YYYY-12-31 instead of 1st of month til end of month.

**Comment by kdenzel** [comment]
Yes, it is an enormous overhead when i have to reload the events from the server and the current view is month instead of the year.
